### ISSUE TITLE
"Resource not found." for Elsevier EISSN's

### ISSUE THREAD

**Issue by acmccorm** [CrossRef/rest-api-doc#369]
I am trying to get the DOI's for Elsevier journals. For any:
https://api.crossref.org/journals/[ELSEVIER EISSN]/works?select=DOI
the page displays "Resource not found."

Example:
https://api.crossref.org/journals/1873-5584/works?select=DOI
"Resource not found."

I have been looking through old Issue threads but couldn't find an issue with a solution for this problem.

Possible bug? My error?

**Comment by JimHokanson** [comment]
The 1873-5584 doesn't work even without the works endpoint:
https://api.crossref.org/journals/1873-5584

filter:issn:1873-5584 for /works should also work but returns 0 matches

https://api.crossref.org/works?filter=issn%3A1873-5584

If we get an article from that ISSN:
https://api.crossref.org/works/10.1016/j.apsusc.2018.04.237

We see only 0169-4332, which is the print ISSN

Other publishers do publish both, for example see:
https://api.crossref.org/works/10.1007/s003450050072

So in summary it seems the issue is that Elsevier doesn't place their EISSNs in their works (or at least doesn't for the example you provided)

**Comment by pdavis8** [comment]
Hello,
That is correct @JimHokanson, this title record on our database does not currently have the eISSN attached to it. I will contact the publisher and ask them to add that metadata to the record.
Thanks

**Comment by pdavis8** [comment]
I contacted the publisher who have now replied stating that the publication does not have an eISSN which is why there is not one attached. Although there are cases where the metadata has just been missed or not included.
